hbase_sql/src/test/java/com/splicemachine/hbase/PhysicalDeletionIT.java

```diff
@@ -84,14 +94,14 @@ public void testPhysicalDelete() throws Exception {
         Thread.sleep(2000); // wait for commit markers to be written
 
         try (Connection connection = ConnectionFactory.createConnection(HConfiguration.unwrapDelegate())) {
-            methodWatcher.execute("CALL SYSCS_UTIL.SYSCS_FLUSH_TABLE('PHYSICALDELETIONIT','A')");
-            methodWatcher.execute("CALL SYSCS_UTIL.SET_PURGE_DELETED_ROWS('PHYSICALDELETIONIT','A',true)");
-            methodWatcher.execute("CALL SYSCS_UTIL.SYSCS_PERFORM_MAJOR_COMPACTION_ON_TABLE('PHYSICALDELETIONIT','A')");
+            methodWatcher.execute("CALL SYSCS_UTIL.SYSCS_FLUSH_TABLE('" + SCHEMA1 + "','A')");
+            methodWatcher.execute("CALL SYSCS_UTIL.SET_PURGE_DELETED_ROWS('" + SCHEMA1 + "','A',true)");
+            methodWatcher.execute("CALL SYSCS_UTIL.SYSCS_PERFORM_MAJOR_COMPACTION_ON_TABLE('" + SCHEMA1 + "','A')");
```

Comment:
Can you also add a test that makes sure that purging does/doesn't happen during flush when we specify a retention period